filter/metrics_parser.go

```diff
@@ -71,6 +72,23 @@ func ParseMetric(input []byte) (*ParsedMetric, error) {
 	return parsedMetric, nil
 }
 
+func restoreMetricStringByNameAndLabels(name string, labels map[string]string) string {
+	var result = ""
+	keys := make([]string, 0, len(labels))
+	for key := range labels {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+
+	result += name
+
+	for _, key := range keys {
+		result += fmt.Sprintf(";%s=%s", key, labels[key])
```

Comment:
Maybe better use strings.Builder for concatenate strings?

Reply:
moved to strings.Builder